lms/services/d2l_api/client.py

```diff
@@ -117,6 +117,32 @@ def course_table_of_contents(self, org_unit) -> List[dict]:
         )
         return D2LTableOfContentsSchema(response).parse().get("modules", [])
 
+    def public_url(self, org_unit, file_id) -> str:
```

Comment:
Service method for getting a public URL for a D2L file. So far this is all very familiar :smile: